packages/smooth_app/lib/pages/product/autocomplete.dart

```diff
@@ -30,34 +34,43 @@ class AutocompleteOptions<T extends Object> extends StatelessWidget {
       child: Material(
         elevation: 4.0,
         child: ConstrainedBox(
-          constraints: BoxConstraints(maxHeight: maxOptionsHeight),
-          child: ListView.builder(
-            padding: EdgeInsets.zero,
-            shrinkWrap: true,
-            itemCount: options.length,
-            itemBuilder: (BuildContext context, int index) {
-              final T option = options.elementAt(index);
-              return InkWell(
-                onTap: () {
-                  onSelected(option);
-                },
-                child: Builder(builder: (BuildContext context) {
-                  final bool highlight =
-                      AutocompleteHighlightedOption.of(context) == index;
-                  if (highlight) {
-                    SchedulerBinding.instance
-                        .addPostFrameCallback((Duration timeStamp) {
-                      Scrollable.ensureVisible(context, alignment: 0.5);
-                    });
-                  }
-                  return Container(
-                    color: highlight ? Theme.of(context).focusColor : null,
-                    padding: const EdgeInsets.all(LARGE_SPACE),
-                    child: Text(displayStringForOption(option)),
-                  );
-                }),
-              );
-            },
+          constraints: BoxConstraints(
+            maxHeight: maxOptionsHeight,
+            maxWidth: maxOptionsWidth,
+            minWidth: 100.0,
+          ),
+          child: Scrollbar(
+            child: ListView.separated(
```

Comment:
`Scrollbar` AND `ListView`? What about removing the `Scrollbar` you've added?

Reply:
The ListView only shows a list a scrolling item without a Scrollbar.
If we want one, we have to manually add it

Reply:
The scrolling is embedded. The confusion probably comes from somewhere else:

https://api.flutter.dev/flutter/widgets/ListView-class.html
> ## ListView class
> A scrollable list of widgets arranged linearly.

Reply:
OK then, `ListView` is already scrollable but you want to add a visual scrollbar, am I right?

Reply:
Yes, exactly